Assets/MRTK/StandardAssets/EditorUtilities/OnLoadUtilities.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    [InitializeOnLoad]
+    public static class OnLoadUtilities
```

Comment:
Does this need to be public? I don't think anything inside it is.

Reply:
you are right, i will change that. thx!